theHook

```diff
@@ -2,7 +2,7 @@
 
 <?php
 
-require_once 'vendor/autoload.php';
+require_once __DIR___ . '/vendor/autoload.php';
```

Comment:
Hey, thanks for that.
I think you have a typo here, 3 underscores after DIR. Can you fix it?

Reply:
![image](https://user-images.githubusercontent.com/199095/59097840-3a847580-8927-11e9-9c70-d683b03bbe38.png)

you do not need to bother authors for such tiny changes